pulp_rpm/app/migrations/0040_disttree_digest.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.9 on 2021-12-20 22:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rpm', '0039_rpmalternatecontentsource'),
```

Comment:
Needs rebase

Reply:
You mean that you want to release this fix before bringing back the ACS support?

Reply:
I think we decided we *have* to do that if we want it to be backportable to Sat with Pulp 3.15